### PR TITLE
Provenance metadata research

### PR DESCRIPTION
**Pull request by Flurb**

### CODE REVIEW

File: DATABASE_MANAGEMENT.md [hunk 1]
```diff
@@ -70,4 +70,55 @@ A single database is restricted to 2 billion nodes (also, see [BaseX Statistics]
 A node in this case is an XML node like an element, attribute, text, etc.
 
 ### Sources
-http://www.adamretter.org.uk/presentations/restxq_mugl_20120308.pdf
\ No newline at end of file
+http://www.adamretter.org.uk/presentations/restxq_mugl_20120308.pdf
+
+## Provenance Overview
```
Comment (Sander3003) [inline DATABASE_MANAGEMENT.md:75]: Where do you want to store de prov information in the database? In a seperate database?

**Review by Sander3003 — CHANGES_REQUESTED** [review]
Please add the idea to store the prov data.

**Review by Sander3003 — APPROVED** [review]
Looks good to me.

File: DATABASE_MANAGEMENT.md [hunk 2]
```diff
@@ -70,4 +70,77 @@ A single database is restricted to 2 billion nodes (also, see [BaseX Statistics]
...
+This is enough for the basic usage that we need. For every version (every edit for example), a new hItem will be added to the SCL.
+This way, we don't need to extend the SCL by default. We can just use the standard.
+
+### Final Thoughts
+The best way is to use the History section of a SCL file.
+It's flexible enough for our basic needs, so we got the version, the when, the who, etc.
+
+In the future it's possible that we need to need something extra, for example an origin CIM file from conversion.
+For this we can use the W3C PROV-XML section.
+In this case, we can extend the History item with so-called 'private' data, which will be the W3C PROV-XML data.
```
[the remark this answers is not in the document]
Comment (Flurb) [reply DATABASE_MANAGEMENT.md:143]: I'll add!

**Pull request merged**